### PR TITLE
bump source-file-secure

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
source-file was updated to a new version

Since we need it in cloud, we have to bump the secure version too

## How
- Make the secure version follow the version from the standard source file
- Fix https://github.com/airbytehq/airbyte/pull/14488/ to make that optional

### CODE REVIEW

File: airbyte-integrations/connectors/source-file/source_file/client.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def _open(self, binary):
             return smart_open.open(uri, transport_params=transport_params, mode=mode)
         elif storage in ("https://", "http://"):
             transport_params = None
-            if self._provider["user_agent"]:
+            if "user_agent" in self._provider and self._provider["user_agent"]:
```
Comment (ChristopheDuong) [inline airbyte-integrations/connectors/source-file/source_file/client.py:120]: seems like we need to make that optional (not required)

**Review by Phlair — APPROVED** [review]

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**